components/enterprise-contract/kustomization.yaml

```diff
@@ -18,7 +18,7 @@ configMapGenerator:
 - name: ec-defaults
   namespace: enterprise-contract-service
   literals:
-  - verify_ec_task_bundle=quay.io/hacbs-contract/ec-task-bundle:a98163a138491ba91ffce4e1f1853e3a52943f1e
+  - verify_ec_task_bundle=quay.io/hacbs-contract/ec-task-bundle:741b8a1fd9a3e1019e494d5136000aefde10c590@sha256:eb0cd5e3978c372bc82a5e0ab34d7d4d9c74ef1d18d6ef424ea15d1c0e881d44
```

Comment:
Looking at https://quay.io/repository/hacbs-contract/ec-cli?tab=tags&tag=latest it seems right to me.

Reply:
wait that's not it...

Reply:
https://quay.io/repository/hacbs-contract/ec-task-bundle?tab=tags&tag=latest

Reply:
Second link looks right to me:
![image](https://user-images.githubusercontent.com/5272931/208760722-984280f8-bf70-4e16-abc8-ad5f89270979.png)